string object_type
int64 cobot_id
float64 deadline